src/action/wallet.js

```diff
@@ -403,6 +403,7 @@ class WalletAction {
       this._nav.goWait();
       await this._grpc.sendUnlockerCommand('UnlockWallet', {
         walletPassword: toBuffer(walletPassword),
+        recoveryWindow: this._store.wallet.restoring ? 250 : 0,
```

Comment:
Perhaps I'm missing something, but won't `recoveryWindow ` always be `0`since `store.wallet.restoring` is never set to true?